pipelines/snippets.yml

```
@@ -52,94 +52,78 @@ steps:
+    git config user.name "Microsoft Graph DevX Tooling"
   displayName: 'Git: set user config'
+  workingDirectory: '$(Build.SourcesDirectory)/microsoft-graph-docs'
+
+- task: UseDotNet@2
+  displayName: 'Install dotnet core SDK'
   inputs:
-    targetType: inline
-    workingDirectory: '$(Build.SourcesDirectory)/microsoft-graph-docs'
-    script: |
-      git config user.email "GraphTooling@service.microsoft.com"
-      git config user.name "Microsoft Graph DevX Tooling"
+    version: 5.x
 
 - task: DotNetCoreCLI@2
   displayName: 'Build snippet generator'
   inputs:
     command: 'build'
-    projects: 'microsoft-graph-explorer-api\CodeSnippetsReflection.App\CodeSnippetsReflection.App.csproj'
+    projects: 'microsoft-graph-explorer-api/CodeSnippetsReflection.App/CodeSnippetsReflection.App.csproj'
     arguments: '--configuration $(buildConfiguration)'
 
-- task: NuGetToolInstaller@1
-  displayName: 'Install Nuget.exe'
-
-- task: NuGetCommand@2
-  displayName: 'Restore packages for apidoctor'
-  inputs:
-    restoreSolution: '$(apidoctorSolution)'
-
-- task: VSBuild@1
-  displayName: 'Build apidoctor'
+- task: DotNetCoreCLI@2
+  displayName: 'Restore packages for APIDoctor'
   inputs:
-    solution: '$(apidoctorSolution)'
-    platform: 'Any CPU'
-    configuration: '$(buildConfiguration)'
+    command: 'restore'
+    projects: '$(Build.SourcesDirectory)/$(apidoctorProjects)'
 
-- task: PowerShell@2
-  displayName: 'Generate snippets'
+- task: DotNetCoreCLI@2
+  displayName: 'Build APIDoctor'
   inputs:
-    workingDirectory: microsoft-graph-docs
-    targetType: 'inline'
-    script: |
-      # release folder can change based on .NET core version, so search recursively in bin folder
-      $snippetGeneratorPath = (Get-ChildItem $env:Build_SourcesDirectory\microsoft-graph-explorer-api\CodeSnippetsReflection.App\bin\Release *App.exe -Recurse).FullName
-      Write-Host "Path to snippet generator tool: $snippetGeneratorPath"
+    command: 'build'
+    projects: '$(Build.SourcesDirectory)/$(apidoctorProjects)'
+    arguments: '--configuration $(buildConfiguration)'
 
-      $apidoctorPath = (Get-ChildItem $env:Build_SourcesDirectory\apidoctor\ApiDoctor.Console\bin\Release apidoc.exe -Recurse).FullName
-      Write-Host "Path to apidoctor tool: $apidoctorPath"
+- pwsh: |
+    # release folder can change based on .NET core version, so search recursively in bin folder
+    $snippetGeneratorPath = (Get-ChildItem $env:BUILD_SOURCESDIRECTORY/microsoft-graph-explorer-api/CodeSnippetsReflection.App/bin/Release *App -Recurse).FullName
+    Write-Host "Path to snippet generator tool: $snippetGeneratorPath"
 
-      & $apidoctorPath generate-snippets --ignore-warnings --path . --snippet-generator-path $snippetGeneratorPath --lang $(snippetLanguages) --git-path "C:\Program Files\Git\bin\git.exe"
+    $apidoctorPath = (Get-ChildItem $env:BUILD_SOURCESDIRECTORY/apidoctor/ApiDoctor.Console/bin/Release/net5.0 apidoc -Recurse).FullName
```

Given that we have `-Recurse`, I assume we can safely remove `net5.0` to make it work for future dotnet releases without a code change.

```suggestion
    $apidoctorPath = (Get-ChildItem $env:BUILD_SOURCESDIRECTORY/apidoctor/ApiDoctor.Console/bin/Release apidoc -Recurse).FullName
```